### PR TITLE
MRG: Improve Brain UX

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This PR follows https://github.com/mne-tools/mne-python/pull/8730#issuecomment-767010268 and will also implement other ideas to improve the `Brain` API.

ToDo:

- [x] Show the time label when the sliders are not visible (toggled off)
- [x] Shortcuts to move forward/backward in time
- [x] Shortcuts to rotate the brain along X/Y axis
- [x] Add all keyboard shorcuts in docstring

### CODE REVIEW

**Comment by GuillaumeFavelier** [comment]
@jasmainak  you may be interested in this one :)

**Comment by larsoner** [comment]
For the rotations it's important to clarify if the rotations occur along the x/y axis directions defined by the viewer, or the x/y defined by the brain/MRI. In other words, if I have a left hemi lateral view, and I press "up" a few times to tilt the brain along the y axis so that it "leans back" away from the camera, what happens when I press "left"? Does it rotate about the brain's z axis, or about the z axis of the viewer? This is similar to thinking about trackball (always relative to the viewer) versus terrain (always relative to a standard Z direction of the data).

**Comment by jasmainak** [comment]
Awesome, I'm going to give it a try later today!

**Comment by jasmainak** [comment]
Okay so the time label looks excellent. I checked and it works also with the movie

What do you think of removing the legend in the time course plot and showing the vertex number as a tooltip or in a small panel below (same way that matplotlib shows coordinates)? I don't know if the MNI coordinates are actually useful? If they are, I'd suggest moving them to a panel below and you can get the info when mouseover-ing the time course

Also GFP plot appears really flat in pretty much any plot. Is it actually useful? I presume this is basically standard deviation across all source points?  The flatness may also be due to the plot being really stretched out. One idea is to make the time course panel half in width and adding some info information and/or buttons on the other half. Or other kinds of plots like SNR ... I don't know

Also I have been playing around with the label extraction. It's a nice feature! But unfortunately the labels you get with the default parcellations are quite big and don't often align with the brain activity. In MNE-analyze, you can draw a label with the mouse cursor and look at the average time course in that label. Is this a feature folks would be interested in? I can raise a separate issue for it if so.

File: mne/viz/backends/_utils.py [hunk 1]
```diff
@@ -66,6 +66,12 @@ def _alpha_blend_background(ctable, background_color):
     return (use_table * alphas) + background_color * (1 - alphas)
 
 
+def _clamp(value, rng):
+    value = rng[1] if value > rng[1] else value
+    value = rng[0] if value < rng[0] else value
+    return value
```
Comment (larsoner) [inline mne/viz/backends/_utils.py:72]: no need for this, `np.clip` does this for you

**Review by larsoner — APPROVED** [review]
Pushed a commit to change the keypress step size and directions to match mne_analyze, will merge once CIs come back happy!

**Review by agramfort — APPROVED** [review]

**Comment by larsoner** [comment]
Thanks @GuillaumeFavelier !

**Comment by jasmainak** [comment]
Fantastic, thanks @GuillaumeFavelier !

**Comment by jasmainak** [comment]
@GuillaumeFavelier I just gave this a try and found the following problems:

- if I zoom in to the brain and then try to use the arrow buttons to turn the brain, it zooms out. I think this should not happen
- when navigating in time, if I go beyond the allowed xlim, then it is not possible to come back. It's a bit of a nitpick but it's probably easy to fix that you don't allow the user to go beyond the allowed xlim

**Pull request merged**